src/codeu/chat/server/Controller.java

```diff
@@ -140,6 +177,11 @@ public ConversationHeader newConversation(Uuid id, String title, Uuid owner, Tim
       LOG.info("Conversation added: " + id);
     }
 
+    //Log should load before adding new commands.
```

Comment:
Well, this is mainly explaining the cause of the "If" statement. We only want to store this in the `storedLogCommands` ArrayList after the log has finished loading.